api/lib/controllers/meeting.js

```diff
@@ -265,7 +265,29 @@ module.exports = {
   async index(req, res) {
     const subject_email = req.credentials.user.email;
     const subject_id = req.credentials.sub;
-    const { limit = 0, skip = 0 } = req.query; // Thanks tom
+    const {
+      limit = 0,
+      skip = 0,
+      name = '',
+      owners = []
+    } = req.query;
+
+    const pipelineFilters = [];
+
```

Comment:
If you add an ownerLookup like this:
```js
const ownerLookup = [
  {
    $lookup: {
      from: "users",
      localField: "owner_id",
      foreignField: "_id",
      pipeline: [
        {
          $project: {
            email: 1,
            username: 1,
          },
        },
      ],
      as: "owner",
    },
  },
  {
    $project: {
      name: 1,
      owner: { $arrayElemAt: ["$owner", 0] },
      date: 1,
      status: 1,
    },
  },
];
```
After the pipeline filters you will have the email of the owner for display in the filters.

The resulting docs look like this:
```js
{
  _id: new ObjectId("636576ade17b8d1bb434a2ed"),
  name: 'Stark - Dooley Acquisition',
  date: 2022-11-05T12:55:59.354Z,
  status: 'draft',
  owner: {
    email: 'brian@brian.com',
    username: 'brian'
  }
}
```